config/default/kustomization.yaml

```diff
@@ -12,21 +12,20 @@ namePrefix: etcd-druid-
 #commonLabels:
 #  someName: someValue
 
-bases:
-- ../crd
-- ../rbac
-- ../manager
+resources:
+  - ../crd
+  - ../rbac
+  - ../manager
 
 patches:
-- manager_image_patch.yaml
+  - manager_image_patch.yaml
```

Comment:
Shouldn't this be a list in patches. Why do we need this intendation?

Reply:
Thats was default indentation for yaml configured in vscode. And we use same accross repos. I didn't modified it explicitly. And irrespective of that, i do prefer list to be indent, though yaml allows unindented list.